randovania/games/dread/json_data/Artaria.txt

```diff
@@ -3754,11 +3721,15 @@ Extra - asset_id: collision_camera_056
   * Layers: default
   * Open Passage to Water Tunnel under Map/Dock to Waterfall
   > Door to Map Station
-      Gravity Suit or After Artaria - Water Reservoir Blob
+      Gravity Suit or After Artaria - Water Reservoir Blob or Perform WBJ
   > Right of Rotatable
       All of the following:
-          Gravity Suit
-          Space Jump or Simple IBJ
+          Any of the following:
+              Gravity Suit
+              After Artaria - Water Reservoir Blob and Perform WBJ
+          Any of the following:
+              Simple IBJ or Use Spin Boost
+              Flash Shift and Walljump (Intermediate)
```

Comment:
Can add Speed Boost here

Reply:
This is true, but it's more than just that due to door rando